### PR TITLE
Update preset configuration for simplified usage

### LINKED ISSUE #202: [SPEC] Update preset configuration for simplified usage
## Overview
We need to re-organize and update the `config` directory for easier user onboarding and setup. Users should be able to quickly view the config directory to understand what strategies they want to support, the available supported networks and the configuration details required for each. 

I propose the following directory structure within `config` for preset configurations. You can see `8sided-evm-bridge/` for all the initial networks the relayer should support. 

| Folder | Desc |
|---|---|
| full-support | Contain the configurations necessary for the relayer to support the `SignatureBridge`, `VAnchor`, `Mixer`, data proposing, signature relaying, and data querying for all supported networks.  Basically cover everything in `exclusive-strategies` and exclude `development`|
| development | Contain the configurations for local evm, and substrate development  |
| exclusive-strategies | Contain all the relayer strategies that can be run independently or exclusively  |
| mixer | Contain the configurations for supporting the mixer |
| 8sided-evm-bridge | Contain the configurations necessary to support a specific bridge (e.g. ETH-8sided-evm-bridge)  |
| signature-relaying | Contain the configurations necessary to only propose data and relay signatures  |
| private-tx-relaying | Contain the configurations necessary to only relay transactions privately  |
| data-querying | Contain the configurations necessary to only enable data queries  |

### Example directory organization:
```
config/
├─ full-support/
├─ exclusive-strategies/
│  ├─ 8sided-evm-bridge/
│  │  ├─ rinkeby.toml
│  │  ├─ goerli.toml
│  │  ├─ optimism.toml
│  │  ├─ harmony.toml
│  │  ├─ arbitrum.toml
│  │  ├─ mumbia.toml
│  │  ├─ fuji.toml
│  │  ├─ aurora.toml
│  ├─ mixer (on eggnet)/
│  ├─ private-tx-relaying/
│  ├─ signature-relaying/
│  ├─ data-querying/
├─ development/
│  ├─ local-substrate/
│  ├─ local-evm/
```

### Expected Behaviour 

We want it to be incredibly simple for users to make use of these preset configs. The user should only have to set their `.env`
with their private keys, run docker with their preferred preset config and have a fully functioning relayer configured to their preference. 

### Tasks

- [ ] Update VAnchor contract addresses once deployed
- [ ] Add `.toml` files to each specified directory and supported networks
- [ ] Add comments to each field with adequate explanation / purpose  
- [ ] Test each preset config to ensure it works as expected 

### Related Tasks

- https://github.com/webb-tools/relayer/issues/195
- https://github.com/webb-tools/relayer/pull/200

### PR DESCRIPTION
**Pull request by dharjeezy**
## Summary of changes
- Include different configurations in different directories

### Reference issue to close (if applicable)
- Closes #202 

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: config/full-support/eggnet-mixer/substrate.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+port = 9955
+
+[substrate.webbeggnet]
+enabled=true
+chain-id=1080
+http-endpoint = "https://standalone.webb.tools"
+ws-endpoint = "wss://standalone.webb.tools"
+runtime = "WebbProtocol"
+suri = "//Alice"
```
Comment (dutterbutter) [inline config/full-support/eggnet-mixer/substrate.toml:9]: This should be a private value no?

File: config/full-support/testnet-evm-bridge/.env.example [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+POLYGON_TESTNET_WSS_URL=
+POLYGON_TESTNET_PRIVATE_KEY=
+
+RINKEBY_HTTPS_URL=
+RINKEBY_WSS_URL=
+RINKEBY_PRIVATE_KEY=
+
+ROPSTEN_HTTPS_URL=
+ROPSTEN_WSS_URL=
+ROPSTEN_PRIVATE_KEY=
```
Comment (dutterbutter) [inline config/full-support/testnet-evm-bridge/.env.example:27]: Please add Kovan network

**Review by drewstone — COMMENTED** [review]
What is the meaning of changing the name of files to `testnet-eth-like-bridge`? @dharjeezy

File: config/exclusive-strategies/testnet-erc20-bridge/moonbase.toml [hunk 3]
```diff
@@ -58,10 +58,10 @@ withdraw-config = { withdraw-fee-percentage = 0, withdraw-gaslimit = "0x350000"
...
-  { chain = "goerli", chain-id = "5", address = "0x4e22da303c403daaf4653d3d9d63ef009bae89a6" },
-  { chain = "polygontestnet", chain-id = "80001", address = "0xe6b075ecc4ccbc6e66569b1a2984cc47e88ee246" },
-  { chain = "ropsten", chain-id = "3", address = "0x35295fbb71273b84f66e70b8e341d408150dcaf9" },
-  { chain = "optimismtestnet", chain-id = "420", address = "0x12f2c4a1469b035e4459539e38ae68bc4dd5ba07"}
+  { type="Evm", chain-id = 42161, address = "0x91a9a1e76fa609f6ba8fcd718a60b030678765ad" },
+  { type="Evm", chain-id = 4, address = "0x7ae23a95881bf8ab86174e89bd79199f398d19bf" },
+  { type="Evm", chain-id = 5, address = "0x4e22da303c403daaf4653d3d9d63ef009bae89a6" },
+  { type="Evm", chain-id = 80001, address = "0xe6b075ecc4ccbc6e66569b1a2984cc47e88ee246" },
+  { type="Evm", chain-id = 3, address = "0x35295fbb71273b84f66e70b8e341d408150dcaf9" },
+  {type="Evm", chain-id = 420, address = "0x12f2c4a1469b035e4459539e38ae68bc4dd5ba07"}
```
Comment (drewstone) [inline config/exclusive-strategies/testnet-erc20-bridge/moonbase.toml:66]:
```suggestion
  { type="Evm", chain-id = 420, address = "0x12f2c4a1469b035e4459539e38ae68bc4dd5ba07"}
```

File: config/full-support/testnet-erc20-bridge/goerli.toml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+withdraw-config = { withdraw-fee-percentage = 0, withdraw-gaslimit = "0x350000" }
+# Entries for this anchor contract's connected edges.
+# These fields are used to determine the generation of AnchorUpdate proposals
+linked-anchors = [
+  { type="Evm", chain-id = 421613, address = "0x91a9a1e76fa609f6ba8fcd718a60b030678765ad" },
+  { type="Evm", chain-id = 3, address = "0x35295fbb71273b84f66e70b8e341d408150dcaf9" },
+  { type="Evm", chain-id = 80001, address = "0xe6b075ecc4ccbc6e66569b1a2984cc47e88ee246" },
+  { type="Evm", chain-id = 420, address = "0x12f2c4a1469b035e4459539e38ae68bc4dd5ba07" },
+  { type="Evm", chain-id = 4, address = "0x7ae23a95881bf8ab86174e89bd79199f398d19bf"},
+  { type="Evm",chain-id = 1287, address = "0xc6b43568f0c39e3a68b597a3bb54a7b9e4308bf3"}
```
Comment (drewstone) [inline config/full-support/testnet-erc20-bridge/goerli.toml:66]:
```suggestion
  { type="Evm", chain-id = 1287, address = "0xc6b43568f0c39e3a68b597a3bb54a7b9e4308bf3"}
```

**Pull request closed without merging**